modules/console/pages/config/security/authentication.adoc

```diff
@@ -55,88 +61,74 @@ flowchart TD
     D --> L[Git]
 ....
 
-Upon successful login, Redpanda Console creates a secure session by generating a JSON Web Token (JWT). Signed with `authentication.jwtSigningKey` and optionally stored as a secure cookie, the JWT encapsulates your credentials (an OIDC ID token or username/password) to authenticate API requests.
-
-=== OIDC authentication
-
-With OIDC, users are redirected to an external identity provider, such as Google, for authentication. After login, the provider issues an ID token that is encapsulated in a JWT. This token is used in the following ways:
-
-* **Kafka authentication:**
-  The ID token is mapped to SASL-OAUTHBEARER, allowing Kafka to authenticate requests under the user's identity.
-
-* **HTTP API authentication:**
-  The ID token is used in header-based authentication (`Authorization: Bearer <id-token>`) in the Schema Registry and Admin API.
-
-Redpanda must be configured with OIDC authentication for its APIs. The OIDC configurations across these APIs must match. Using a consistent identity provider avoids authentication mismatches and ensures seamless integration.
-
-=== Basic authentication
-
-For basic authentication, the user submits a username and password directly to Redpanda Console. These credentials are validated and encapsulated within a JWT, which is then used as follows:
+== Prerequisites
 
-* **Kafka authentication:**
-  The provided credentials are mapped to SASL-SCRAM, ensuring that Kafka operations execute with the correct user identity.
+* You must have at least one superuser in Redpanda.
+* The authentication method used in Redpanda Console must match the configuration of the Kafka API:
 
-* **HTTP API authentication:**
-  The JWT supports HTTP Basic authentication for other APIs.
+**If using OIDC:**
+You must have SASL/OAUTHBEARER authentication configured for the Kafka API.
 
-== Prerequisites
+**If using basic authentication:**
```

Comment:
These sub-bullets are not correctly aligned:

![image](https://github.com/user-attachments/assets/b5562245-1f96-4ea5-85e9-ddaae40ca47b)